scripts/renderItems.js

```diff
@@ -0,0 +1,117 @@
+const windowBody = document.querySelector(".window__container__body");
+const binBlock = `  <div class="explorer__window">
+                <div class="explorer__window--left">
+                  <ul class="explorer__window--left--mydocs">
+                    <li></li>
+                    <li><img src="./images/sys32.png" alt=""> System32 </li>
+                    <li> <img src="./images/familypic.png" alt="" /> Drunk Selfie</li>
+                    <li><img src="./images/worddoc.png" alt=""> Assignment </li>
+
+                  </ul>
+                </div>
+                <div class="explorer__window--right"></div>
+              </div>`;
+const documentBlock = ` <div class="explorer__window">
+  <div class="explorer__window--left">
+    <ul class="explorer__window--left--mydocs">
+      <li id="puppy" class="imgClick">
+        <img src="./images/familypic.png" alt="" /> Puppy
+      </li>
+      <li id="ny" class="imgClick">
+        <img src="./images/familypic.png" alt="" /> New York
+      </li>
+      <li id="hw" class="imgClick">
+        <img src="./images/worddoc.png" alt="" /> Homework
+      </li>
+    </ul>
+  </div>
+  <div class="explorer__window--right">
+    <img class="previewContainer" id="preview" src="./images/ny.jpeg" alt="" />
+  </div>
+</div>`;
+
+export const renderItems = (id) => {
+  switch (id) {
+    case "bin":
+      clearWindow();
+      windowBody.innerHTML = binBlock;
+      customElement(windowBody, "div", "", "explorerDiv");
+
+      break;
+    case "mycomputer":
+      clearWindow();
+      break;
+    case "ie":
+      clearWindow();
+      customElement(
+        windowBody,
+        "iframe",
+        "https://askjeeves.net/",
+        "windowIframe"
+      );
+      break;
+    case ("mydocs", "documents"):
```

Comment:
Every single item has `clearWindow()` except for this one
There's also a lot more lines under this case, it seems to do more things
Without knowing intimately what this function is doing, this indicates to me that perhaps you're making an assumption that this case is similar to the others when in fact it is not and should have it's own function